sources/packages/api/src-sql/EducationPrograms/Alter-education-programs.sql

```diff
@@ -0,0 +1,159 @@
+--Drop the NOT NULL constraint on program_description
+ALTER TABLE
+    sims.education_programs
+ALTER COLUMN
+    program_description DROP NOT NULL;
+
+--Changing the data type of has_minimun_age as it is one of entrance requirement now.
+ALTER TABLE
+    sims.education_programs
+ALTER COLUMN
+    has_minimun_age TYPE BOOLEAN USING CASE
+        has_minimun_age
+        WHEN 'yes' THEN TRUE
+        ELSE FALSE
+    END;
+
+-- Add institution_program_code to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN IF NOT EXISTS institution_program_code VARCHAR(50);
+
+COMMENT ON COLUMN sims.education_programs.institution_program_code IS 'Institution program code of the given program';
+
+-- Add min_hours_week to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN IF NOT EXISTS min_hours_week VARCHAR(50);
+
+COMMENT ON COLUMN sims.education_programs.min_hours_week IS 'This column represents result for the question, Does this program include a minimum of 20 instructional hours per week? ';
+
+-- Add is_aviation_program to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN IF NOT EXISTS is_aviation_program VARCHAR(50);
+
+COMMENT ON COLUMN sims.education_programs.is_aviation_program IS 'This column determines if the given program is aviation program';
+
+-- Add min_hours_week_avi to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN IF NOT EXISTS min_hours_week_avi VARCHAR(50);
+
+COMMENT ON COLUMN sims.education_programs.min_hours_week_avi IS 'This column determins if an aviation program satisfies minimum hours per week';
+
+-- Add min_high_school to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN IF NOT EXISTS min_high_school BOOLEAN;
+
+COMMENT ON COLUMN sims.education_programs.min_high_school IS 'This column respresents the entrance requirement of minimum grade 12 or equivalent';
+
+-- Add requirements_by_institution to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN IF NOT EXISTS requirements_by_institution BOOLEAN;
+
+COMMENT ON COLUMN sims.education_programs.requirements_by_institution IS 'This column respresents the entrance requirement established by the institution that enable completion of the program study.';
```

Comment:
I like to have sentences with periods in the end, commas, etc. In the past, I asked to do it, but the PRs are not following and I gave up after insisting a bit.
That said, we should at least be consistent if we are going to add the periods, add them always do it or do not at all 😉 
@ann-aot @guru-aot FYI

Reply:
Typo for _respresents_.

Reply:
I'm ok to follow the agreement of period. Sometimes it gets missed out.

Reply:
Period added to comments.